lib/crypto-util.ts

```diff
@@ -0,0 +1,237 @@
+import webcrypto from './get-crypto.js';
+import { curve25519 } from './curve25519.js';
+
+import { RSAddress } from './rs-address.js'
+
+
+export namespace CryptoUtil {
+    export namespace Converters {
+        export function strToHex(str: string): string {
+            let result = '';
+            strToBytes(str).forEach(c => result += c.toString(16));
+            return result;
+        }
+
+        export function strToBytes(str: string): number[] {
+            const result = [];

```

Comment:
You could be a bit more functional and do `Array.from(str).map(c => c.charCodeAt(0))`

Reply:
Will try, thanks

Reply:
Done.